web/ui/static/js/prom_console.js

```diff
@@ -138,10 +138,17 @@ PromConsole.TimeControl.prototype._setHash = function() {
 
 PromConsole.TimeControl._initialValues = function() {
   var hash = window.location.hash;
+  var values;
   if (hash.indexOf('#pctc') === 0) {
-    return JSON.parse(decodeURIComponent(hash.substring(5)));
+    values = JSON.parse(decodeURIComponent(hash.substring(5)));
+  } else {
+    values = {duration: 3600, endTime: 0};
+  }
+  if (values.endTime == 0) {
```

Comment:
Ugh. Yay for implicitness... it'll also catch `[]` and who knows what in JS too :)